packages/react-reconciler/src/ReactFiberLane.new.js

```diff
@@ -85,38 +85,54 @@ export const SyncLane: Lane = /*                        */ 0b0000000000000000000
 export const SyncBatchedLane: Lane = /*                 */ 0b0000000000000000000000000000010;
 
 export const InputDiscreteHydrationLane: Lane = /*      */ 0b0000000000000000000000000000100;
-const InputDiscreteLanes: Lanes = /*                    */ 0b0000000000000000000000000011000;
-
-const InputContinuousHydrationLane: Lane = /*           */ 0b0000000000000000000000000100000;
-const InputContinuousLanes: Lanes = /*                  */ 0b0000000000000000000000011000000;
-
-export const DefaultHydrationLane: Lane = /*            */ 0b0000000000000000000000100000000;
-export const DefaultLanes: Lanes = /*                   */ 0b0000000000000000000111000000000;
-
-const TransitionHydrationLane: Lane = /*                */ 0b0000000000000000001000000000000;
-const TransitionLanes: Lanes = /*                       */ 0b0000000001111111110000000000000;
-const FirstTransitionLane: Lane = /*                    */ 0b0000000000000000010000000000000;
-
-const RetryLanes: Lanes = /*                            */ 0b0000011110000000000000000000000;
-
-const FirstRetryLane: Lanes = /*                        */ 0b0000000010000000000000000000000;
-export const SomeRetryLane: Lane = FirstRetryLane;
-
-export const SelectiveHydrationLane: Lane = /*          */ 0b0000100000000000000000000000000;
-
-const NonIdleLanes = /*                                 */ 0b0000111111111111111111111111111;
-
-export const IdleHydrationLane: Lane = /*               */ 0b0001000000000000000000000000000;
-const IdleLanes: Lanes = /*                             */ 0b0110000000000000000000000000000;
+const InputDiscreteLane: Lanes = /*                     */ 0b0000000000000000000000000001000;
+
+const InputContinuousHydrationLane: Lane = /*           */ 0b0000000000000000000000000010000;
+const InputContinuousLane: Lanes = /*                   */ 0b0000000000000000000000000100000;
+
+export const DefaultHydrationLane: Lane = /*            */ 0b0000000000000000000000001000000;
+export const DefaultLane: Lanes = /*                    */ 0b0000000000000000000000010000000;
+
+const TransitionHydrationLane: Lane = /*                */ 0b0000000000000000000000100000000;
+const TransitionLanes: Lanes = /*                       */ 0b0000000000000111111111000000000;
```

Comment:
Shouldn't this have 14 bits on?

Reply:
Dunno what you mean, I don't make mistakes